tools/lkl/lib/virtio.c

```diff
@@ -549,8 +547,8 @@ int virtio_dev_setup(struct virtio_dev *dev, int queues, int num_max)
 
 	if (!lkl_is_running()) {
 		avail = sizeof(lkl_virtio_devs) - (devs - lkl_virtio_devs);
-		num_bytes = snprintf(devs, avail, " virtio_mmio.device=%d@0x%"PRIxPTR":%d",
-				     mmio_size, (uintptr_t) dev->base, dev->irq);
+		num_bytes = snprintf(devs, avail, " virtio_mmio.device=%d@%p:%d",
```

Comment:
IIRC %p does not work correctly for mingw. The mingw tests seem to confirm this.